### DISCUSSION TITLE
Produced kwh

### DISCUSSION THREAD

**Discussion by Flisan2010** [WillCodeForCats/solaredge-modbus-multi#31] [General]
Hi!
I can find imported kwh and exported kwh via meter, but not produced kwh via inverter.
Is i possible to get this information (totalt produced energi) from the inverter?
//Flisan

**Comment by WillCodeForCats** [comment]
Yes it's in there. List all your sensors.

**Comment by Flisan2010** [comment]
Thanks!
Apologies i found I1_ac_energy_kwh.
Have a nice sunday!
